understanding/20/pause-stop-hide.html

```diff
@@ -1,107 +1,85 @@
          animations, real-time games, and scrolling stock tickers. "Auto-updating" refers to
          content that updates or disappears based on a preset time interval. Common time-based
-         content includes audio, automatically updated weather information, news, stock price
+         content includes automatically updated weather information, news, stock price
          updates, and auto-advancing presentations and messages. The requirements for moving,
          blinking and scrolling content and for auto-updating content are the same except that:
       </p>
-      
+
       <ul>
-         
-         <li>authors have the option of providing the user with a means to control the frequency
-            of updates when content is auto-updating and
-         </li>
-         
-         <li>there is no five second exception for auto-updating since it makes little sense to
-            auto-update for a few seconds and then stop
-         </li>
-         
+         <li>authors have the option of providing the user with a means to control the frequency of updates when content is auto-updating and</li>
+         <li>there is no five second exception for auto-updating since it makes little sense to auto-update for a few seconds and then stop</li>
       </ul>
-      
+
       <p>Content that moves or auto-updates can be a barrier to anyone who has trouble reading
          stationary text quickly as well as anyone who has trouble tracking moving objects.
          It can also cause problems for screen readers.
       </p>
-      
+
       <p>Moving content can also be a severe distraction for some people. Certain groups, particularly
          those with attention deficit disorders, find blinking content distracting, making
          it difficult for them to concentrate on other parts of the Web page. Five seconds
          was chosen because it is long enough to get a user's attention, but not so long that
          a user cannot wait out the distraction if necessary to use the page.
       </p>
-      
+
       <p>Content that is paused can either resume in real-time or continue playing from the
          point in the presentation where the user left off.
       </p>
-      
+
       <ul>
-         
          <li>
-            
             <p>Pausing and resuming where the user left off is best for users who want to pause to
                read content and works best when the content is not associated with a real-time event
                or status.
             </p>
-            
             <div class="note">
-               
-               <p>See 
-                  <a href="timing-adjustable">2.2.1: Timing Adjustable</a> for additional requirements related to time-limits for reading.
-               </p>
-               
+               <p>See <a href="timing-adjustable">2.2.1: Timing Adjustable</a> for additional requirements related to time-limits for reading.</p>
```

Comment:
> for reading

and importantly for providing input via forms - maybe add that?

Reply:
added "and interactions"